backend/api/oauthApi.test.js

```diff
@@ -180,22 +180,22 @@ describe('oathApi tests', () => {
     })
   })
 
-  describe('addUserRole', () => {
+  describe('addUserRoles', () => {
```

Comment:
testing got missed out when new method added, so changing to use that one instead now